site/docs/hosting/vps.md

```diff
@@ -0,0 +1,121 @@
+# VPS
+
+A virtual private server, mostly known as VPS is a virtual machine running in the cloud with its users having the full control of its filesystem.
+There are tons of advantages in using a VPS over things like Heroku and microservices to run your bots.
+
+In this guide, you'll learn about various methods of running your bot in a VPS, keeping it online 24/7, making it run automatically when your VPS boots and restart on crashes.
+
+## Methods of Hosting
+
+- [systemd](#systemd)
+
+---
+
+# systemd
```

Comment:
Please check out the deploy preview. This structure does not render well. The link on the side are missing.